go.mod

```diff
@@ -187,8 +187,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	helm.sh/helm/v3 v3.16.1 // indirect
-	istio.io/api v1.24.0-alpha.0.0.20241025181149-a1abc531eea1 // indirect
-	istio.io/client-go v1.24.0-alpha.0.0.20241025181449-b7de4ee9463e // indirect
+	istio.io/api v1.24.0-rc.0.0.20241101200753-9397ebf09c3a // indirect
+	istio.io/client-go v1.24.0-rc.0.0.20241101201152-acb34243979a // indirect
```

Comment:
A more recent version has been already updated as part of https://github.com/istio/istio.io/commit/89260256c972e721debda5a48d82bbf998fbd628#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R13